src/components/RoomStats/RoomStats.tsx

```diff
@@ -0,0 +1,80 @@
+import { useConnectionState, useLocalParticipant, useRemoteParticipants } from '@livekit/components-react'
+import PeopleIcon from '@mui/icons-material/People'
+import SignalCellularAltIcon from '@mui/icons-material/SignalCellularAlt'
+import WifiIcon from '@mui/icons-material/Wifi'
+import { ConnectionQuality, ConnectionState } from 'livekit-client'
+import { useTranslation } from '../../modules/translation'
+import { RoomStatsProps } from './RoomStats.types'
+import { StatItem, StatLabel, StatValue, StatsContainer, StatsHeader, StatsList, StatsTitle, StatusWrapper } from './RoomStats.styled'
+
+export function RoomStats({ isStreamer }: RoomStatsProps) {
+  const { t } = useTranslation()
+  const connectionState = useConnectionState()
+  const { localParticipant } = useLocalParticipant()
+  const remoteParticipants = useRemoteParticipants()
+
+  // For streamer: only count viewers (remote participants)
+  // For viewer: only count viewers (remote participants, excluding the viewer themselves)
+  const totalParticipants = remoteParticipants.length
+  const connectionQuality = localParticipant?.connectionQuality || 'unknown'
+
+  const getConnectionString = (state: ConnectionState) => {
+    switch (state) {
+      case ConnectionState.Connected:
+        return t('room_stats.connected')
+      case ConnectionState.Connecting:
+      case ConnectionState.Reconnecting:
+        return t('room_stats.connecting')
+      case ConnectionState.Disconnected:
+        return t('room_stats.disconnected')
+      default:
+        return t('room_stats.unknown')
+    }
+  }
+
+  const getQualityString = (quality: ConnectionQuality | 'unknown'): string => {
+    switch (quality) {
+      case ConnectionQuality.Excellent:
+        return t('room_stats.excellent')
+      case ConnectionQuality.Good:
+        return t('room_stats.good')
+      case ConnectionQuality.Poor:
+        return t('room_stats.poor')
+      default:
+        return t('room_stats.unknown')
+    }
+  }
```

Comment:
Here we can wrap those functions inside a useCallback or move it into a util function, what do you think?